core/templates/dev/head/app.js

```diff
@@ -471,7 +471,8 @@ oppia.factory('oppiaDatetimeFormatter', ['$filter', function($filter) {
       var date = new Date(millisSinceEpoch);
       if (date.toLocaleDateString() === new Date().toLocaleDateString()) {
         // The replace function removes 'seconds' from the time returned.
-        return date.toLocaleTimeString().replace(/:\d\d /, ' ');
+        return date.toLocaleTimeString('en-US',
```

Comment:
I tried doing it without that, but then safari and firefox were inconsistent with chrome. Safari and Firefox showed time as "11:13:56 AM" while chrome showed it as "11:13:56". Chrome should have also shown "AM/PM" as I had set "hour12" to be true, but it didn't. I don't know how exactly this is working but this made it consistent across the three browsers.